platform/qt/QMapboxGLConfig.cmake.in

```diff
@@ -0,0 +1,13 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(Qt5Gui @REQUIRED_QT_VERSION@)
+find_dependency(Qt5Network @REQUIRED_QT_VERSION@)
+
+if(MBGL_WITH_QT_HEADLESS OR NOT MBGL_WITH_QT_LIB_ONLY)
```

Comment:
This include file leads to rather unexpected issue for the projects using this cmake. Namely, when Mapbox GL Native is compiled and installed with `MBGL_WITH_QT_LIB_ONLY=ON`, the section below is still used in the projects including this cmake as a part of finding Mapbox: `find_package(QMapboxGL REQUIRED)`. So, platforms without OpenGL, would fail to find Qt5OpenGL.

Somehow, the original choice made while compiling Mapbox GL should stick in generated configs.